testsupport/wait/awaitility.go

```diff
@@ -384,7 +386,8 @@ func (a *Awaitility) DeletePods(criteria ...client.ListOption) error {
 	if err != nil {
 		return err
 	}
-	for _, p := range pods.Items {
+	for i := range pods.Items {
+		p := pods.Items[i] // avoids the 'G601: Implicit memory aliasing in for loop. (gosec)' problem
 		if err := a.Client.Delete(context.TODO(), &p); err != nil {
```

Comment:
the same for this

Reply:
reverted in bc57e8a, using a `nolint:gosec` commend instead